libs/source/Socket.cpp

```diff
@@ -28,23 +28,13 @@ Socket::Socket(Socket::type stype) {
 }
 
 int Socket::setBufferSizes(int sndbuf, int rcvbuf) {
-  int ok{true};
-  int res{0};
   if (sndbuf) {
-    if ((res = setSockOpt(SO_SNDBUF, &sndbuf, sizeof(sndbuf))) < 0) {
-      ok = false;
-    }
+    setSockOpt(SO_SNDBUF, &sndbuf, sizeof(sndbuf));
```

Comment:
I don't really understand what's going on here. Why was this class simplified?

Reply:
because this particular call cannot fail and always returns ok.